gatsby-config.js

```diff
@@ -5,6 +5,33 @@ module.exports = {
     author: `@gatsbyjs`,
   },
   plugins: [
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        // The property ID; the tracking code won't be generated without it
+        trackingId: process.env.GOOGLE_ANALYTICS_TRACKING_ID,
```

Comment:
Do not use env at this time. Hardcode first.

Reply:
done.